tests/vcs/git/GitCommitBearTest.py

```diff
@@ -125,6 +127,64 @@ def test_empty_message(self):
                          [])
         self.assert_no_msgs()
 
+    def test_revert_checks(self):
+        self.git_commit('Test commit')
+        commit_hash, _ = run_shell_command('git rev-parse HEAD')
+        self.git_commit('Revert "Test commit"\n\n'
+                        'Abcdef blablabla commit ' + str(commit_hash) + '\n\n'
+                        'Reason for the revert commit is explained here.\n')
+        self.assertEqual(self.run_uut(),
+                         ['Invalid revert commit.'])
+        self.assert_no_msgs()
+
+        self.git_commit('Test commit1\n\n'
+                        'This is body of test commit1\n')
+        commit_hash, _ = run_shell_command('git rev-parse HEAD')
+
+        self.run_git_command('revert', 'head')
```

Comment:
These revert commits are redundant since you are already making another commit on top of these, so Commit Bear is never run on them
(Similarly on line 179)